imap_processing/swe/l2/swe_l2.py

```diff
@@ -1,33 +1,99 @@
 """SWE L2 processing module."""
 
+import numpy as np
 import numpy.typing as npt
+import xarray as xr
 
-from imap_processing.swe.l1b.swe_l1b_science import read_lookup_table
+from imap_processing.swe.utils.swe_utils import read_lookup_table
 
-# TODO: add this to instrument status summary
+# TODO: add these to instrument status summary
 ENERGY_CONVERSION_FACTOR = 4.75
```

Comment:
Should these constants be in a swe constants.py type file? Fine if they're only used here, just wanted to ask